documentation/source/simulator_support.rst

```diff
@@ -256,6 +256,33 @@ Support is preliminary.
 Noteworthy is that despite GHDL being a VHDL simulator, it implements the VPI interface.
 
 
+.. _sim-ghdl-waveforms:
+
+Waveforms
+---------
+
+To get waveforms in VCD format, set the :make:var:`SIM_ARGS` option to ``--vcd=anyname.vcd``,
+for example in a Makefile:
+
+.. code-block:: make
+
+    SIM_ARGS+=--vcd=anyname.vcd
+
+The option can be set on the command line, as shown in the following example.
+
+.. code-block:: bash
+
+    SIM_ARGS=--vcd=anyname.vhd make SIM=ghdl
+
+A VCD file named ``anyname.vcd`` will be generated in the current directory.
+
+The :make:var:`SIM_ARGS` option allows access to all GHDL Command Line Run (-r)
+options listed in :ref:`ghdl:simulation_options`. This includes other waveform
+generation options including the compressed Fast Signal Trace (FST) format and
+the GHDL Wave File (GHW) which supports any VHDL type. For more information
+see :ref:`ghdl:export_waves`.
```

Comment:
I would reword this slightly. The first sentence is kind of redundant, and I would use better name for the cross-refs. Riff on this if you want.

```suggestion
:make:var:`SIM_ARGS` can also be used to pass command line arguments related to other waveform formats supported by GHDL like :ref:`Fast Signal Trace (FST) <ghdl:export_waves>`, which is compressed; or :ref:`GHDL Wave File (GHW) <ghdl:export_waves>`, which supports any VHDL type. 
```